akasha-master/src/arcasha/assistant/ui.html

```
@@ -0,0 +1,1272 @@
+          <div class="set-row" id="customModelRow" style="display:none">
+            <label data-i18n="set.customModel">カスタムモデル名</label>
+            <input type="text" id="setCustomModel" placeholder="例: deepseek-reasoner / my-model-name" />
+            <div class="desc" data-i18n="set.customModelDesc">オーケストレーションの推論ノードがこのモデル名で API を呼びます。</div>
+          </div>
+        </div>
+        <div class="set-card">
+          <h3>🎛 オーケストレーション</h3>
+          <div class="set-row">
+            <label data-i18n="set.orchestration">参加モデル数</label>
+            <div style="display:flex;align-items:center;gap:12px">
+              <input type="range" id="setCount" min="1" max="50" step="1" value="2" />
```

HTMLHint (1.9.2)

[warning] No matching [ label ] tag found.

(input-requires-label)